examples/angular_view_engine/src/lib/shorten/BUILD.bazel

```diff
@@ -6,6 +6,7 @@ package(default_visibility = ["//:__subpackages__"])
 # with a main field to show how to create a first-party npm lib with a package.json
 ts_library(
     name = "shorten",
+    package_name = "@bazel/shorten",
```

Comment:
when should we drop the view engine example? is that still used by anyone in Angular land? @alan-agius4 ?

Reply:
good question; I kept it green in this PR but I'm not against removing it as it is tied to angular v9